<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedualClass</class>
 <widget class="QMainWindow" name="SchedualClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>664</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Schedual</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>312</y>
      <width>481</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>详细信息</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit"/>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>311</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>日程时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>日程标题</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>日程内容</string>
     </property>
    </column>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前时间：</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="setting">
    <property name="title">
     <string>系统设定</string>
    </property>
    <addaction name="OpenStart"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="TimeSetting">
    <property name="title">
     <string>提醒设置</string>
    </property>
    <addaction name="TimeAlarm"/>
   </widget>
   <widget class="QMenu" name="MusicSetting">
    <property name="title">
     <string>音乐设定</string>
    </property>
    <addaction name="Music"/>
   </widget>
   <widget class="QMenu" name="PersonalNote">
    <property name="title">
     <string>个人记事</string>
    </property>
    <addaction name="NoteBook"/>
    <addaction name="TeleBook"/>
   </widget>
   <widget class="QMenu" name="HelpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="Help"/>
    <addaction name="separator"/>
    <addaction name="About"/>
   </widget>
   <widget class="QMenu" name="Translation">
    <property name="title">
     <string>数据同步</string>
    </property>
    <addaction name="synchro"/>
   </widget>
   <addaction name="setting"/>
   <addaction name="TimeSetting"/>
   <addaction name="MusicSetting"/>
   <addaction name="PersonalNote"/>
   <addaction name="Translation"/>
   <addaction name="HelpMenu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="Help">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>关于本系统</string>
   </property>
  </action>
  <action name="OpenStart">
   <property name="text">
    <string>开机启动</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="TimeAlarm">
   <property name="text">
    <string>定时提醒</string>
   </property>
  </action>
  <action name="FestivalAlarm">
   <property name="text">
    <string>节日提醒</string>
   </property>
  </action>
  <action name="Music">
   <property name="text">
    <string>音乐设定</string>
   </property>
  </action>
  <action name="NoteBook">
   <property name="text">
    <string>个人日记</string>
   </property>
  </action>
  <action name="TeleBook">
   <property name="text">
    <string>通讯录</string>
   </property>
  </action>
  <action name="PasswdBook">
   <property name="text">
    <string>个人密码本</string>
   </property>
  </action>
  <action name="synchro">
   <property name="text">
    <string>蓝牙手机同步</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="schedual.qrc"/>
 </resources>
 <connections/>
</ui>
